graylog2-web-interface/src/components/common/CountBadge.jsx

```diff
@@ -0,0 +1,19 @@
+// @flow strict
+import * as React from 'react';
+import styled, { type StyledComponent } from 'styled-components';
+
+import type { ThemeInterface } from 'theme';
+
+const Badge: StyledComponent<{}, ThemeInterface, HTMLSpanElement> = styled.span(({ theme }) => `
+  background-color: ${theme.colors.variant.light.info};
+`);
+
+type Props = {
+  children: React.Node;
+};
+
+const CountBadge = ({ children }: Props) => (
```

Comment:
This Component is a lie. It does not count anything. It is a colored badge. And you cannt even change the color!!!! Perhaps you should rename it? Do we really need this?